analyzers/tests/SonarAnalyzer.UnitTest/TestCases/NormalizeStringsToUppercase.CSharp11.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Globalization;
+
+namespace Tests.Diagnostics
+{
+    class Program
+    {
+        void StringLowerCalls()
+        {
+            var score = 100;
+            var s1 = $"The score: {score} is {score switch
+            {
+                > 0 => "Positive",
+                < 0 => "Negative",
+                _ => "Zero",
+            }}".ToLower(); // Compliant
+
+            var s2 = $"The score: {score} is {score switch
+            {
+                > 0 => "Positive",
+                < 0 => "Negative",
+                _ => "Zero",
+            }}".ToLowerInvariant(); // Noncompliant
```

Comment:
Noncompliant? According to StringOperationWithoutCulture.cs test cases, it is the other way around: `ToLowerInvariant` is compliant and `ToLower` is NonCompliant.